.github/workflows/ci.yaml

```diff
@@ -35,92 +31,6 @@ jobs:
       # FIXME: Add yamllint problem matcher
       - run: yamllint -s .
 
-  crev:
```

Comment:
Figured while cleaning up other things it might be good to remove steps that aren't currently benefiting our CI.
While crev or something like it sounds good it's too much of a lift at this time to try and review all transitive dependencies, we're not in a place and likely not going to be in a place where this check becomes actionable. Becuase of that this check is just noise at this time.